src/main/java/io/excaliburfrc/robot/subsystems/Shooter.java

```diff
@@ -36,6 +36,8 @@
   private double velocity = 0;
 
   public Shooter() {
+    leader.setInverted(true);
```

Comment:
I think this should be inside the `ValidateREVCAN` call. And it should definitely be after the `restoreFactoryDefaults()` call...

Reply:
it returns void while the REVCAN function only gets methods that return REVError